packages/turf-boolean-contains/index.ts

```diff
@@ -219,10 +219,14 @@ export function isPolyInPoly(
     return false;
   }
 
-  const coords = getGeom(feature2).coordinates;
+  const geom1 = getGeom(feature1);
+  const geom2 = getGeom(feature2);
+  if (!geom1 || !geom2) return false;
```

Comment:
Perhaps all these nulls checks should be more precise `if (geom1 === null || geom2 === null)`.